code/modules/projectiles/guns/energy.dm

```diff
@@ -135,23 +136,18 @@
 	damage_falloff_mult = 0.5
```

Comment:
The tesla gun is a child of the parent lasgun, so if no stats are inputted here, they'd fill up by the parent. And duplicating stats would add a little bloat.

Reply:
okay if you say so